index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="styles.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kevin-Koome-Portfolio</title>
+</head>
+<body>
+    <div class="container">
+            <nav class="Top-Navigation">
+                <div class="Item-Navigation">
+                    <a href="#" class="Contact">Contact</a>
+                    <a href="#" class="About">About</a>
+                    <a href="#" class="Portfolio">Portfolio</a>
+                </div>
+            </nav>
+            <div class="Name">
+                <p class="Koome">Kevin Koome</p>
```

Comment:
- [ ] Replace `p` with `a` tag. The Logo usually is a link that redirects to the homepage.